src/components/Balance/Balance.res

```diff
@@ -0,0 +1,33 @@
+%%raw(`import './balance-component-styles.css';`)
+
+@react.component
+let make = (~allTransactions: array<Transaction.structure>) => {
+  let allBalances = BalanceUtils.getAllBalances(allTransactions)
+
+  <div className="balance-component">
+    <div className="balance-total">
+      <div className="balance-total-content">
+        <p> {BalanceStatic.topHeading} </p>
+        <h1 id="balance-amount-display">
+          {TypeConversions.float_to_RString(allBalances.balance)}
+        </h1>
+      </div>
+    </div>
+
+    <div className="balance-combined">
+      <div className="balance-combined-inner-white">
+        <div className="balance-combined-income">
+          <p> {BalanceStatic.subHeadingLeft} </p>
+          <p className="income-amount"> {TypeConversions.float_to_RString(allBalances.income)} </p>
+        </div>
```

Comment:
Since this code is being used twice, create a module in the same file and use that here